### DISCUSSION TITLE
Header A1 Variants

### DISCUSSION THREAD

**Discussion by ibnsina77** [system-ui/theme-ui#1755] [Q&A] [answered]
would you please tell me, what should the variant part of the code do?

Should e.g. the variant: 'styles.header' pull some stylisation automatically, or that has to be implemented manually?
In my case, I do not get any styles from the variants.

`<header
  sx={{
    display: 'flex',
    alignItems: 'center',
    variant: 'styles.header',
  }}>
  <Link
    to="/"
    sx={{
      variant: 'styles.navlink',
      p: 2,
    }}>
    Hello
  </Link>
  <div sx={{ mx: 'auto' }} />
  <Link
    to="/blog"
    sx={{
      variant: 'styles.navlink',
      p: 2,
    }}>
    Blog
  </Link>
  <Link
    to="/about"
    sx={{
      variant: 'styles.navlink',
      p: 2,
    }}>
    About
  </Link>
</header>`

Thank you

**Comment by hasparus** [comment, marked as answer]
hey @tarikmiljkovic :wave: 
Could you show us your theme definition? `styles.header` doesn't have any default styling if you don't define it.

**Reply by ibnsina77** [reply]
Thank you. Exactly.
Besides, I forgot to add 

/** @jsxImportSource theme-ui */
